### ISSUE TITLE
winget client is not discoverable in the Microsoft Store

### ISSUE THREAD

**Issue by Stephanevg** [microsoft/winget-cli#4927]
### Brief description of your issue

[I opened a bug earlier ](https://github.com/microsoft/winget-cli/issues/4924#event-15013824035) which I think got closed a little bit to quickly, as only 2 points out of the 3 have been fixed (Thank you for the fast fix! 👍🏻 👍🏻 !!)

If you try to search for 'winget' or even 'app installer' in the Microsoft store, you get only 'weird' results back.
It is **impossible** to find winget in the microsoft sotre EXCEPT if you use the link you guys provided.

I also tried to searched for `App installer`  or `Windows package manager`. didnt work. (It is also a bit confusing that there is absolutley no mention of winget in the details of 'App Installer'.

In other words, winget is __not__ discovorable on the microsoft store - which according to your documentation, is the `recommended` way of installing it.

### Steps to reproduce

open the microsoft store, and search for 'winget'

### Expected behavior

We should have the winget client FIRST, then other results that are of the same 'nature'.

### Actual behavior

a lot of different results get returned back. But NONE of them is the actually winget client.

### Environment

windows 11

**Comment by denelon** [comment]
WinGet is included in the "App Installer" which is a system component. System components aren't "searchable" in the Microsoft Store:

![Image](https://github.com/user-attachments/assets/381da01f-2698-4f79-81d0-7b8327967916)

Getting the latest version should happen automatically with the Microsoft Store (as long as it's not disabled).

You can manually trigger the update in the Microsoft Store with "Get updates".

We do need to update the documentation.
